### PR TITLE
FEAT: Added signup page(without functionality)

### PR DESCRIPTION
**Pull request by zoyatanweer**
- added signup page (without functionality)
- added route for it in "App.js" file

File to check :
-  components/ signup/ Signup.jsx

### CODE REVIEW

**Review by dev-prabhat — APPROVED** [review]

**Review by purvasharmadev — APPROVED** [review]
looks good to go.

**Review by Muzammil-khan710 — APPROVED** [review]
Clean code 
good to merge !

File: src/components/signup/Signup.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/* Signup */
+.signup-page {
+  display: flex;
+  flex-direction: row;
```
Comment (iRohitGaur) [inline src/components/signup/Signup.css:4]: direction is row by default. Remove this line unless you are trying to override a column direction

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Pull request merged**